engines/mohawk/riven.cpp

```diff
@@ -151,6 +152,8 @@ Common::Error MohawkEngine_Riven::run() {
 		_cursor = new MacCursorManager("Riven");
 
 	initVars();
+	initKeymap();
```

Comment:
It seems to me `initKeymap` is meant to be called by the base code when starting the engine, not by the engine itself.

Reply:
I'm confused. When you refer to base code, are you talking about MohawkEngine?
If this is the case, then IMHO it's a bad idea to have a common keymap for all the mohawk subengines, as each one has different hotkeys

Reply:
Sorry for being unclear, I'm referring to this: https://github.com/scummvm/scummvm/blob/5bda62b12a7216215a3b52c5509f39fd25b13e2f/base/main.cpp#L281

Reply:
You are quite right. Fixed.